Sources/llbuild2fx/SpawnProcess.swift

```diff
@@ -317,7 +317,7 @@ public struct ProcessInputTreeID: FXSingleDataIDValue, FXTreeID {
     }
 }
 
-public struct ProcessOutputTreeID: FXSingleDataIDValue, FXTreeID {
+public struct ProcessOutputTreeID: FXSingleDataIDValue, FXTreeID, Codable {
```

Comment:
Adding codable to an already existing FXValue doesn't sound right.  These are already being encoded and decoded from the CAS.